modules/living_spaces_event/living_spaces_event.module

```diff
@@ -284,9 +284,12 @@ function living_spaces_event_living_spaces_event_presave(LivingSpaceEventInterfa
             if (!$event->get('space')->isEmpty() && !living_spaces_event_check_user_status($event->id(), $member->id())) {
               /** @var \Drupal\group\Entity\Group $space */
               $space = $event->get('space')->entity;
-
+              $circle_ids = array_column($space->get('circles')->getValue(), 'target_id');
```

Comment:
Living space event module doesn't have a dependency to the living space circles module. So, there is a case when we don't have a circles field.